src/adapters/index.ts

```diff
@@ -0,0 +1,19 @@
+import { SupabaseClient } from "@supabase/supabase-js";
+import { Context } from "../types/context";
+import { Access } from "./supabase/helpers/tables/access";
+import { User } from "./supabase/helpers/tables/user";
+import { Label } from "./supabase/helpers/tables/label";
+import { Locations } from "./supabase/helpers/tables/locations";
+import { Super } from "./supabase/helpers/tables/super";
+
+export function createAdapters(supabaseClient: SupabaseClient, context: Context) {
+  return {
+    supabase: {
```

Comment:
Doesn't seem right that the plugin that handles assistive pricing is initializing a supabase client. 

Assistive pricing should only look up the price label in relation to the price and priority.  

All of the relevant values should be available in the local config ymls of the org/repo